### ISSUE TITLE
Thumbnail spacing disappears

### ISSUE THREAD

**Issue by MistressAshai** [usagirei/PSO2_SymbolArt_ThumbnailHandler#1]
When in a folder, sometimes the file spacing disappears.
This mainly seems to happen when I scroll through the folder.

![github](https://user-images.githubusercontent.com/8222603/85013926-3c783b80-b11a-11ea-9360-70240c6e45d3.png)

**Comment by usagirei** [comment]
Yeah, i've had that happen, but i think its an windows explorer problem:

Microsoft's own guidelines state that if a thumbnail of size 256 is requested, for the handler to provide a non-padded, maximum available sized image with a maximum dimension of 256 in its original aspect ratio (so i'd provide a 256x128 image for a 192x96 .sar), then windows explorer is responsible for padding it.

~Additionally, it's also stated to not add adornments like the drop-shadows pictures have around them that windows explorer is also responsible for adding it, but i've yet to see this happen.~

I've figured out the drop shadow part at least.

**Comment by usagirei** [comment]
@MistressAshai try updating with the new release (v1.0.0.31), after i added the thumbnail frame registry entry, i haven't had the icon collapse on itself, even after agressively scrolling through my cache dir

EDIT:

Nvm, when editing the readme to include a new screenshot with the shadows, the icons were small-ish, it may be intentional design, I can trigger it by changing the icon types from tiles to large, medium icons, etc
